src/ai/AIBase.h

```diff
@@ -28,47 +28,59 @@ class GameClientPlayer;
 class GlobalGameSettings;
 class GameClientPlayerList;
 
-
-
-/// Basisklasse für sämtliche KI-Spieler
+/// Base class for all AI player. The AIBase class provides an interface which may be implemented by different AI
+/// players. 
 class AIBase
 {
     protected:
-        /// Eigene PlayerID, die der KI-Spieler wissen sollte, z.B. wenn er die Karte untersucht
+
+		/// Unique player ID an AI player needs, e.g. to inspect the map. 
         const unsigned char playerid;
-        /// Verweis auf die Spielwelt, um entsprechend Informationen daraus zu erhalten
+
+		/// Reference to the game world to get information for strategic planning
         const GameWorldBase& gwb;
-        /// Verweis auf den eigenen GameClientPlayer, d.h. die Wirtschaft, um daraus entsprechend Informationen zu gewinnen
+
+		/// Reference to its own GameClientPlayer to get additional information. e.g. about economy
         const GameClientPlayer& player;
-        /// Verweis auf etwaige andere Spieler, bspw. um deren Bündnisse zu überprüfen etc.
+
+		/// Reference to a list of all player, e.g. to check for possible alliances
         const GameClientPlayerList& players;
-        /// Queue der GameCommands, die noch bearbeitet werden müssen
-        std::vector<gc::GameCommandPtr> gcs;
-        /// Stärke der KI
+
+		/// Queue of game comments to process
+		std::vector<gc::GameCommandPtr> gcs;
+
+		/// AI difficulty level
         const AI::Level level;
-        /// Abstrahiertes Interfaces, leitet Befehle weiter an
-        AIInterface* aii;
+
+		/// Abstract interface to forward commands (???)
+		AIInterface* aii;
 
     public:
 
+		/// Creates a new instance of AIBase initializes the members with the specified 
+		/// information (game world, client player, player list, etc.).
         AIBase(const unsigned char playerid, const GameWorldBase& gwb, const GameClientPlayer& player,

```

Comment:
Constructors don't really need comments. Unless they do something specific. E.g. When you have a deserializing ctor or one that does something specific.